oeis.py

```diff
@@ -237,6 +237,33 @@ def A000041(start, limit):
     return [len(partitions(n)) for n in range(start, start + limit)]
 
 
+@oeis
+def A001220(start, limit, plot):
+    sequence = []
```

Comment:
This lacks a sequence description, as a docstring (take a look at other sequences).